templates/tfengine/components/iam_members/main.tf

```diff
@@ -0,0 +1,46 @@
+{{- /* Copyright 2021 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */ -}}
```

Comment:
I don't think it's needed to add it between `{{- -}}` block if you do:

```
# Copyright 2021 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

I've seen it in few components, but I think it creates noise because usually, the `-` trimmers are difficult to read.

You may leave it but I guess it'll be healthy if we keep a standard License between components.

Reply:
This follows the existing formatting of the other components. It is not good idea to make this one different from the existing ones, unless we make a change to every existing component which is out of the scope of this PR.